manuscript.md

```diff
@@ -14,6 +14,37 @@
 
 ## What is currently enabling our ability to predict network structure?
 
+The acquisition of biodiversity and environmental data has tremendously 
+increased over the past decades thanks to the rise of citizen science 
+[@Dickinson2010CitSci] and of novel technology [@Stephenson2020TecAdv], 
+including wireless sensors [@Porter2005WirSen], DNA monitoring 
+[@Creer2016EcoSF], and satellite remote sensing [@Skidmore2015AgrBio; @Lausch2016LinEar]. 
+Standard practices in data integration and quality control [@Kissling2018BuiEss] and in 
```

Comment:
We could maybe also include standardised sampling protocols e.g. plant functional traits [Pérez-Harguindeguy et. al., (2016)](https://www.publish.csiro.au/bt/pdf/BT12225) and metadata e.g. [DarwinCore](https://www.tdwg.org/) as this also helps promote interoperability making it easier to integrate multiple data sources